graphql/src/main/java/com/scalar/db/graphql/server/ServerConfig.java

```diff
@@ -0,0 +1,110 @@
+package com.scalar.db.graphql.server;
+
+import com.google.common.base.Strings;
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.LinkedHashSet;
+import java.util.Properties;
+import java.util.Set;
+import javax.annotation.concurrent.Immutable;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Immutable
+@SuppressFBWarnings("JCIP_FIELD_ISNT_FINAL_IN_IMMUTABLE_CLASS")
+class ServerConfig {
+  public static final String PREFIX = "scalar.db.graphql.";
+  public static final String PORT = PREFIX + "port";
+  public static final String PATH = PREFIX + "path";
+  public static final String NAMESPACES = PREFIX + "namespaces";
+  public static final String GRAPHIQL = PREFIX + "graphiql";
+
+  public static final int DEFAULT_PORT = 8080;
+  public static final boolean DEFAULT_GRAPHIQL = true;
+  private static final String DEFAULT_PATH = "/graphql";
+  private static final Logger LOGGER = LoggerFactory.getLogger(ServerConfig.class);
+  private final Properties props;
+  private int port;
+  private String path;
+  private Set<String> namespaces;
+  private boolean graphiql;
+
+  public ServerConfig(File propertiesFile) throws IOException {
+    this(new FileInputStream(propertiesFile));
+  }
+
+  public ServerConfig(InputStream stream) throws IOException {
+    props = new Properties();
+    props.load(stream);
+    load();
+  }
+
+  public ServerConfig(Properties properties) {
+    props = new Properties();
+    props.putAll(properties);
+    load();
+  }
+
+  private void load() {
+    port = getInt(PORT, DEFAULT_PORT);
+    path = props.getProperty(PATH, DEFAULT_PATH);
+    namespaces = new LinkedHashSet<>(Arrays.asList(getStringArray(NAMESPACES)));
+    graphiql = getBoolean(GRAPHIQL, DEFAULT_GRAPHIQL);
+  }
+
+  private int getInt(String name, int defaultValue) {
```

Comment:
We can use `ConfigUtils` here:
https://github.com/scalar-labs/scalardb/blob/69b435e552be4241e93e41b55c8e31c4a29a681e/core/src/main/java/com/scalar/db/config/ConfigUtils.java

Reply:
Merged `master` to the `graphql` branch, then updated the code to use `ConfigUtils` in 6ad93d9. Thank you!

Reply:
Also, I moved the `ServerConfig` class into the `config` sub-package as with the other XXXConfig files.

Reply:
Ah sorry. Can you please move back the` ServerConfig` class to the `com.scalar.db.graphql.server` package? I'm going to do the same thing in the `server` module.